components/com_users/src/Model/ResetModel.php

```diff
@@ -342,9 +342,38 @@ public function processResetConfirm($data)
 			return false;
 		}
 
+		// Check if the token is expired or not
+		$currDate = strtotime(Factory::getDate());
+		$lastResetDate = strtotime($user->lastResetTime);
+
+		// Formulate the Difference between two dates
+		$diff = abs($currDate - $lastResetDate);
+		$years = floor($diff / (365 * 60 * 60 * 24));
+
+		/*
+		To get the month, subtract it with years and
+		divide the resultant date into
+		total seconds in a month (30 * 60 * 60 * 24)
+		*/
```

Comment:
https://developer.joomla.org/coding-standards/inline-code-comments.html

```suggestion
		/**
		 * To get the month, subtract it with years and
		 * divide the resultant date into
		 * total seconds in a month (30 * 60 * 60 * 24)
		 */
```